fsspec_reference_maker/hdf.py

```diff
@@ -131,12 +135,19 @@ def transfer_attrs(self, h5obj, zobj):
                 zobj.attrs[n] = v
             except TypeError:
                 print(f'Caught TypeError: {n}@{h5obj.name} = {v} ({type(v)})')
+                lggr.exception(' ')
```

Comment:
I agree to prefer logging over the print on the line above, which should be removed/combined with this

Reply:
The print statement was leftover from debugging. Removed.